godot/prefabs/GameController/GameController.cs

```diff
@@ -5,76 +5,77 @@
 
 public partial class GameController : Node
 {
-	public Passenger[] AvailablePassengers { get; private set; } = Array.Empty<Passenger>();
-	private Turtle turtleInstance;
-	private Timer gameTimer;
-	
-	[Export]
-	public float Score { get; private set; } = 0;
+    public Passenger[] AvailablePassengers { get; private set; } = Array.Empty<Passenger>();
+    private Turtle turtleInstance;
+    private Timer gameTimer;
+    
+    [Export]
+    public double Score { get; private set; } = 0;
```

Comment:
something else to consider is something like
```suggestion
    public int Score => (int)Math.Floor(m_score);
    private double m_score = 0.0;
```

Reply:
Nice, good call. Yes int is much better